components/exact/sources/new-account-created/new-account-created.mjs

```diff
@@ -0,0 +1,27 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "exact-new-account-created",
+  name: "New Account Created",
+  description: "Emit new event each time a new account is created. [See the docs](https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=CRMAccounts)",
```

Comment:
```suggestion
  description: "Emit a new event each time a new account is created. [See the docs](https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=CRMAccounts)"
```

Reply:
Changed 'Emit new event' to 'Emit a new event' to follow Google's style guide (use articles)